app/helpers/results_helper.rb

```diff
@@ -9,19 +9,42 @@ def render_field_item(doc)
         doc[:document][doc[:field]].join(", ").html_safe
   end
 
+  # truncate abstract to 450 characters
+  def render_abstract(doc)
+    doc[:document][doc[:field]][0].truncate(450).html_safe
+  end
   ##
   # Render clean faceted link to items in series
   def render_series_facet_link(doc)
-    series = doc[:document][doc[:field]]
-    collection = doc[:document][Solrizer.solr_name("collection", :displayable)].first
-    links_to_series = []
+    series = doc[:document][Solrizer.solr_name("parent_unittitles", :displayable)]
+   
+    coll = doc[:document][Solrizer.solr_name("collection", :displayable)].first
 
-    series.each do |ser|
-      links_to_series << link_to(ser, add_clean_facet_params_and_redirect([series_facet, ser],[collection_facet, collection]))
+    if doc[:document][Solrizer.solr_name("unittitle", :displayable)].nil?
+      item_title =  doc[:document][doc[:field]][0].split(">>").last.html_safe
+    else
+      item_title = doc[:document][Solrizer.solr_name("unittitle", :displayable)][0]
     end
-    [links_to_series].join(" >> ").html_safe
+  
+    title = content_tag(:span,item_title, class: "result_ut") if item_title != coll  # don't get title if it is a collection
```

Comment:
@eshadatta And since you no longer call this function in the case where it's a collection, which makes sense always since a collection never belongs to a series, you can take out the conditionals relating to collections in this function.

You may want to revisit if you can revert this to the original function and just push the collection title to the front on all occasions.